Sources/Danger/GitLabDSL.swift

```diff
@@ -51,7 +51,7 @@ extension GitLab {
             public let createdAt: Date
             public let description: String
             public let dueDate: Date?
-            public let id: Int // swiftlint:disable:this identifier_name
+            public let id: Int 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitLabDSL.swift;
  Line: 54;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)